src/core_plugins/kibana/public/home/components/add_data.js

```diff
@@ -20,7 +21,8 @@ import {
   EuiText,
 } from '@elastic/eui';
 
-export function AddData({ addBasePath, isCloudEnabled }) {
+const apmUiEnabled = chrome.getInjected('apmUiEnabled');
```

Comment:
@nreese Would you rather have `apmUiEnabled` passed in as a prop (like you did with `isCloudEnabled`?)
Injecting `apmUiEnabled` from outside the React cycle is slightly dirty but it feels more wrong to pass it through several layers that don't need it - and it'll be removed in 6.4 anyway (I assume), so it's probably easier if it's just in a single place.

Reply:
I would recommend passing it in as a prop

Reply:
done